stable/influxdb/templates/job.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if .Values.setDefaultUser.enabled -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+  name: {{ template "fullname" . }}-create-user
+  annotations:
+    "helm.sh/hook": post-install
+spec:
+  activeDeadlineSeconds: {{ default 300 .Values.setDefaultUser.activeDeadlineSeconds }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+        release: "{{ .Release.Name }}"
+    spec:
+      containers:
+      - name: {{ template "fullname" . }}-create-user
+        image: "{{ .Values.setDefaultUser.image }}"
+        args:
+          - "-X"
+          - POST
+          - "http://{{ template "fullname" . }}:{{ .Values.config.http.bind_address }}/query"
```

Comment:
Does this need to be `https://{{ template "fullname" . }}.{{ .Release.Namespace }}:{{ .Values.config.http.bind_address }}/query`?